site/standalone/setup.md

```diff
@@ -41,7 +41,11 @@ can restart `flux` and it will continue to use the same key.
 Flux connects to the repository using an SSH key.
 
 ***The SSH key must be configured to have R/W access to the repository***.
- 
+
+More specifically, in the case of standalone Flux, the ssh key must be able to
+create and update tags. E.g. in Gitlab, that means it requires `Maintainer`
+permissions. The `Developer` permission can create tags, but not update them.
+
```

Comment:
It's slightly more graded than that; if you want to be able to do releases through flux (automated, or otherwise), it has to be able to push commits, too.

Reply:
Agree :-)

I was writing up something, but I do not have enough background for those scenario's. Can someone else complete this explanation then ?

Reply:
Hi @squaremo to get this PR not get stale, I could also simply remove the sentence above, so that at least the other aspects of this PR are not lost.

Reply:
Good idea; let's defer that bit.

Reply:
Maybe change it to list the things it must be able to do? Which is:

 - push commits to your chosen branch
 - force push the sync tag